src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -7,6 +7,7 @@
 import numpy as np
 import os
 from pathlib import Path
+import openvino as ov
```

Comment:
@akuporos do we need this import? I see `from openvino import` below, which can be reused if needed

Reply:
no, we don't need it
@inbasperu please, get rid of it